examples/twitter-compose-with-typeahead/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@algolia/autocomplete-example-twitter-compose-with-typeahead",
+  "description": "Autocomplete example powering a Twitter compose box with mentions typeahead.",
+  "version": "1.4.0",
+  "private": true,
+  "license": "MIT",
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test",
+    "eject": "react-scripts eject"
+  },
+  "dependencies": {
+    "@algolia/autocomplete-core": "1.4.0",
+    "@algolia/autocomplete-preset-algolia": "1.4.0",
+    "@algolia/client-search": "4.10.5",
+    "@types/textarea-caret": "3.0.1",
```

Comment:
We can move that to `devDependencies`.

Reply:
Yes, it failed oddly initially but that was probably caused by ESLint's mischief. Handled in 2abafa8a.